File: cytoscape/enron.js
var core = require('./core');

const path = '../samples/Email-Enron.txt';
const avoidDoubles = true;
const uniqueTests = [100, 1000, 10000];
const indexes = [100, 1000, 10000];
const algorithms = ['bfs', 'dfs', 'shortest'];

core.run(path, avoidDoubles, uniqueTests, indexes, algorithms);

File: igraph/enron.py
import core

PATH = '../samples/Email-Enron.txt'
AVOID_DOUBLES = True
UNIQUE_TESTS = [100, 1000, 10000]
INDEXES = [100, 1000, 10000]
ALGORITHMS = ['bfs', 'dfs', 'shortest']

core.run(PATH, AVOID_DOUBLES, UNIQUE_TESTS, INDEXES, ALGORITHMS)